README.md

```diff
@@ -2,9 +2,9 @@
 
 # smtp-client
 
-> Simple, promisified, protocol-based SMTP client for Node.js.
+> Simple promised, protocol-based SMTP client for Node.js.
 
-This is an open source [npm](http://npmjs.com) package from [Node.js](http://nodejs.org). The source code is available on [GitHub](https://github.com/xpepermint/smtp-client) where you can also find our [issue tracker](https://github.com/xpepermint/smtp-client/issues).
+This is an open-source [npm](http://npmjs.com) package from [Node.js](http://nodejs.org). The source code is available on [GitHub](https://github.com/xpepermint/smtp-client) where you can also find our [issue tracker](https://github.com/xpepermint/smtp-client/issues).
```

Comment:
Open-source in this case is used as an adjective, and the correct spelling is in fact "open-source", as suggested.